YIF.Core.Service/Concrete/Services/IoEAdminService.cs

```diff
@@ -195,5 +195,21 @@ public async Task<ResponseApiModel<SpecialtyToInstitutionOfEducationResponseApiM
                 Success = true
             };
         }
+
+        public async Task<ResponseApiModel<DescriptionResponseApiModel>> DeleteIoEModerator(string moderatorId, string userId)
+        {
+            var result = new ResponseApiModel<DescriptionResponseApiModel>();
+            var adminId = (await _institutionOfEducationAdminRepository.GetByUserId(userId)).Id;
+            var moderator = await _ioEModeratorRepository.GetModeratorForAdmin(moderatorId, adminId);
+
```

Comment:
you should drop user's moderator role there

Reply:
ok, I`ll di that!